Makefile

```diff
@@ -33,21 +62,33 @@ test:
 
 # Generate manifests, e.g. CRDs
 manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=deploy/crd
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./..." output:crd:artifacts:config=$(INSTALL_ROOT)/crd
 
 # Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations
 k8s-generate: controller-gen
 	$(CONTROLLER_GEN) paths="./..." object
 
+#
+# Installation of just the CRD
+# Can only be executed as a cluster-admin
+#
 install:
```

Comment:
Could the `install` and `deploy` targets be replaced with the `setup` and `operator` targets respectively?

Reply:
Yes, you're right. That was more to check we have the same understanding. We can keep that PR atomic and have them removed later on. Only the documentation from the README should be impacted.